spec/javascript/components/organisms/StatusHistoryPopover.test.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { act } from 'react-dom/test-utils'
+import { mockManyStatusItems } from '../../mocks/statusItemMock'
+
+import StatusHistoryPopover from '~/components/organisms/StatusHistoryPopover'
+
+const mockGetFieldUpdateComments = jest.fn()
+
+jest.mock('apiService', () => {
+  return {
+    getFieldUpdateComments: async (applicationId) => {
+      mockGetFieldUpdateComments(applicationId)
+
+      return mockManyStatusItems(2)
+    }
+  }
+})
+
+const tick = () => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, 0)
+  })
+}
+
+const getWrapper = () => mount(<StatusHistoryPopover applicationId={'app_id'} />)
+
+describe('StatusHistoryPopover', () => {
+  test('it matches snapshot when open', async () => {
+    const wrapper = getWrapper()
+    act(() => {
+      wrapper.find('button').simulate('click')
+    })
+    await act(async () => await tick)
+    wrapper.update()
+    expect(wrapper).toMatchSnapshot()
+  })
+  test("it loads status history data when data isn't present", async () => {
+    const wrapper = getWrapper()
+    act(() => {
+      wrapper.find('button').simulate('click')
+    })
+    await act(async () => await tick)
+    wrapper.update()
+    expect(mockGetFieldUpdateComments).toHaveBeenCalledTimes(1)
+  })
+  test('it only fetches status history on first click', async () => {
+    const wrapper = getWrapper()
+    act(() => {
+      wrapper.find('button').simulate('click')
+    })
+    await act(async () => await tick)
```

Comment:
Do you mind removing these ticks and seeing if tests still pass? I found (I forget which file it was for) that all I needed was act() and update, no tick needed.

Reply:
just tried removing it even after removing the awaits and asyncs in the StatusHistoryPopover file. Needs the extra tick to resolve the promise and get the status items back to be included in the snapshot